RELEASE-NOTES.md

```diff
@@ -0,0 +1,128 @@
+
+    Gait speed has been demonstrated to be a useful and reliable functional measure of walking ability. When administering the *Timed Walk Task*, patients are allowed to use assistive devices (canes, crutches, walkers).
+
+- **Charts Module**
+
+ *Contributed by [coxy1989](https://github.com/coxy1989) and [Ricardo Sánchez-Sáez](https://github.com/rsanchezsaez).*
+
+ A *Charts module* has been implemented. It features three chart types: a *pie chart* (`ORKPieChartView`), a *line graph chart* (`ORKLineGraphChartView`), and a *discrete graph chart* (`ORKDiscreteGraphChartView`).
+
+ The views in the *Charts module* can be used independently of the rest of *ResearchKit*. It doesn't automatically connect with any other *ResearchKit* module: the developer has to supply the data to be displayed through the views' `dataSources`, which allows for maximum flexibility.
+
+- **Other improvemnets**
+
+ - **Scale Answer Format**
+
+    *Contributed by [Apple Inc](https://github.com/researchkit).*
+
+    *Discrete scales* now support *text choice* labels, and all *scales* support images in place of the minimum and maximum range labels.
+
+ - **Result Predicates**
+
+    *Contributed by [Ricardo Sánchez-Sáez](https://github.com/rsanchezsaez).*
+
+    The predicate-building methods in `ORKResultPredicate` now use the new `ORKResultSelector` class for unequivocally identifying a *question step result* or a *form item result*.
+
+    This eliminates ambiguity when matching results with the same inner scope identifier. For example, a *form item result* can have the same identifier as a *question step result* or as another *form item result* in a different *form step*, and you can now match them separately.
+
+- **General stability and performance improvements**.
+
+## ResearchKit 1.1 Release Notes
+
+*ResearchKit 1.1* supports *iOS* and requires *Xcode 6.3* or newer. The minimum supported *Base SDK* is *8.0*.
+
+- **Navigable Ordered Task**
+
+ *Contributed by [Ricardo Sánchez-Sáez](https://github.com/rsanchezsaez).*
+
+ A new type of *conditional ordered task* (`ORKNavigableOrderedTask`) has been implemented.
+
+ The developer can use the `ORKStepNavigationRule` subclasses to dynamically navigate between the task steps:
+ - `ORKPredicateStepNavigationRule` allows to make conditional jumps by matching previous results (either those of the the ongoing task, or those of any previously stored task result tree). You typically use the class methods in the `ORKResultPredicate` class to match answers in the most commonly used result types.
+ - `ORKDirectStepNavigationRule` provides support for unconditional jumps.
+
+- **New Active Tasks**
+ - **Reaction Time Task**
+
+    *Contributed by [coxy1989](https://github.com/coxy1989).*
+
+    The [*Simple Reaction Time (SRT)*](http://www.cambridgecognition.com/tests/simple-reaction-time-srt) is a test which measures simple reaction time through delivery of a known stimulus to a known location to elicit a known response.
```

Comment:
From our internal edit:
Please replace `The Simple Reaction Time (SRT) is a test` with `The Reaction Time Task is an adaptation of the Simple Reaction Time (SRT) test`.